soundfile.py

```diff
@@ -167,6 +167,38 @@
     __libsndfile_version__ = __libsndfile_version__[len('libsndfile-'):]
 
 
+class SoundFileError(Exception):
```

Comment:
I would prefer moving the exception classes further down, because now they appear first in the API docs.

Reply:
For me it is much more natural to define classes before they are used, especially for exceptions. But I am open for that change.

Reply:
I firmly agree with @mgeier on this one. Code should be structured top-down, i.e. usage-first. I find code much easier to read when high-level concepts are *used* before they are defined, going from the highest-level code to the lower-level implementations.